src/scripts/delete_flow_runs.py

```
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)